# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-reversed-ref.html
# Commit: 4f30d7afdf22
# Full Hash: 4f30d7afdf22c652481513564b60b37053c0948b
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-05-08 04:32:48
# Regressor Bug: 1932150
# File Overlap Count: 2
# Description:
#   Revert "Bug 1932150 - Fix build bustage" for causing build bustages.
#   
#   This reverts commit 0681b181d6a8a48ac460a97dacb7fca91f9989c3.
#   
#   Revert "Bug 1932150 - Fix lint failure" for causing Build bustage.
# ==============================================================================

diff -r 1e64a48e3517 -r 4f30d7afdf22 testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-reversed-ref.html
--- a/testing/web-platform/mozilla/tests/selection/cross-shadow-boundary-slot-reversed-ref.html	Wed May 07 20:11:46 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,5 +0,0 @@
-<!doctype html>
-Start <span id="node2">text2</span><span id="node1">text1</span> End
-<script>
-  window.getSelection().setBaseAndExtent(node2.firstChild, 2, node1.firstChild, 3);
-</script>